net/acme/files/run.sh

```diff
@@ -219,8 +219,6 @@ issue_cert()
         uci set uhttpd.main.cert="$STATE_DIR/${main_domain}/fullchain.cer"
         # commit and reload is in post_checks
     fi
-
-    post_checks
```

Comment:
Can't do this as we actually need to run the post_checks for every cert issued (in case there is more than one section configured).